README.md

```diff
@@ -74,23 +63,20 @@ scram b -j 8
 cd $CMSSW_BASE/src/SKFlatMaker
 source setup.sh
 ```
-
 ### For the Production
 I recommend you to make a clean working directory when we do the central production.
 
 Also, I recommend using lxplus
 
 ```bash
 # scram p -n <directory name> CMSSW <cmssw release>
-# below, assuming we are using tag:Run2Legacy_v2, in cmssw:CMSSW_9_4_10
+# below, assuming we are using tag:Run2Legacy_v3, in cmssw:CMSSW_9_4_13
 export SCRAM_ARCH=slc6_amd64_gcc630
-scram p -n Run2Legacy_v2__CMSSW_9_4_10 CMSSW CMSSW_9_4_10
-cd Run2Legacy_v2__CMSSW_9_4_10/src
+scram p -n Run2Legacy_v3__CMSSW_9_4_13 CMSSW CMSSW_9_4_13
+cd Run2Legacy_v3__CMSSW_9_4_13/src
 cmsenv
 # crab setup
 source /cvmfs/cms.cern.ch/crab3/crab.sh
-# proxy
-voms-proxy-init --voms cms -valid 9999:00
```

Comment:
why remove this?

Reply:
ah I thought proxy will be asked when submitting crab anyway, so not needed here

Reply:
Ok, i agree, but it wont be -valid 9999:00 if that ok with you

Reply:
yep it needs to be only valid during the crab submission so I think it's okay (9999:00 is copied from my .bach_profile lol)